java/system-test/system-test-configuration/src/test/java/sleeper/systemtest/configuration/SystemTestPropertiesTest.java

```diff
@@ -85,6 +87,27 @@ void shouldFindNoUnknownProperties() {
                 .isEmpty();
     }
 
+    @Test
+    void shouldAcceptDefaultValueForSystemPropertyWhenValueNotSet() {
+        // Given
+        SystemTestProperties properties = validProperties();
+
+        // When
+        properties.set(MIN_RANDOM_INT, null);
+
+        // Then
+        assertThat(properties.get(MIN_RANDOM_INT)).isEqualTo("0");
+    }
+
+    @Test
+    void shouldReturnNullValueWhenAttemptingToRetrievePropertyNotSet() {
+        // Given
+        SystemTestProperties properties = validProperties();
+
+        // When / Then
+        assertThat(properties.get(SYSTEM_TEST_REGION)).isNull();
+    }
```

Comment:
Please can we also test getting the value of a property that is set?

Reply:
Addressed